### ISSUE TITLE
[3.x] Importing Formie as a module into JS causes errors for repeater field

### ISSUE THREAD

**Issue by zizther** [verbb/formie#2056]
### Describe the bug

I am using vite to build the JS for the project and I am importing Formie into the JS as shown in the docs. I am not rendering the JS for the form.

The form is multi page and has a repeater field.
When pressing the button to add another row I get the error message `Formie.parseFieldConfig is not a function`
I do not get this message when adding Formie JS to the form by enabling the JS rendering in the template.

Even though there is an error, a new row will be added in the repeater field.

### Steps to reproduce

1. Setup a form with a repeater field
2. Don't render the JS for the form
3. Import the Formie JS into the projects JS file, as shown in the documentation.

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes 
- Custom Form Templates: No

### Craft CMS version

5.4.1

### Plugin version

3.0.5

### Multi-site?

No

**Comment by engram-design** [comment]
Are you able to share (a cut down version of) your JS that's using Formie? I just want to ensure we're on the same page with how you're importing and initializing forms.

**Comment by zizther** [comment]
@engram-design Sure, here is what I have

```
/* Imports */
import { Formie } from '../../../vendor/verbb/formie/src/web/assets/frontend/src/js/formie-lib';

class Forms {
    constructor() {
        document.addEventListener('DOMContentLoaded', (event) => {
            this.initForms();
        });
    }

    initForms() {
        const formieInstance = new Formie();
        formieInstance.initForms();

        document.addEventListener('onFormieInit', (event) => {
            // Fetch the Form Factory once it's been loaded
            let form = event.detail.formie;
            let formId = event.detail.formId;
            let formEl = document.getElementById(formId);

            // Refresh the necessary bits that are statically cached (CSRF inputs, captchas, etc)
            this.update(form, formId);

            if (formEl.getAttribute('data-form-submit-method') !== 'ajax') {
                formEl.addEventListener('onAfterFormieSubmit', (e) => {
                    //e.preventDefault();

                    this.update(form);
                });
            }
        });
    }
    
    update(form, formId) {
        form.refreshForCache(formId);
    }
}

export default new Forms();
```

**Comment by engram-design** [comment]
Ah, that makes sense now. It's because we're assuming incorrectly that there's a `Formie` global variable.

Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.5"`.

**Comment by engram-design** [comment]
Fixed in [3.0.6](https://github.com/verbb/formie/releases/tag/3.0.6).
